src/uk/gov/defra/ffc/Tests.groovy

```diff
@@ -103,62 +138,62 @@ class Tests implements Serializable {
 
   static String getDotNetSonarImageVersion(ctx, projectName) {
     String targetFramework = Version.getCSTargetFramework(ctx, projectName)
-    switch(targetFramework) {
-      case "netcoreapp3.1":
-        return "1.2.3-dotnet3.1"
-      case "net6.0":
-        return "1.3.0-dotnet6.0"
+    switch (targetFramework) {
+      case 'netcoreapp3.1':
+        return '1.2.3-dotnet3.1'
+      case 'net6.0':
+        return '1.3.0-dotnet6.0'
```

Comment:
Just confirming this has been tested for .NET builds?  Jenkins/Groovy combo is very particular about which type of quotes to use for specific tasks.

Just want to make sure the Sonar scanning step identifies the correct image to use.